src/creature_states_mood.c

```diff
@@ -260,7 +260,10 @@ void anger_set_creature_anger_f(struct Thing *creatng, long annoy_lv, AnnoyMotiv
         annoy_lv = 65534;
     }
     TbBool was_angry = ((cctrl->mood_flags & CCMoo_Angry) != 0);
-    cctrl->annoyance_level[reason] = annoy_lv;
+    if ( ((cctrl->spell_flags & CSAfF_MadKilling) == 0) || (annoy_lv > cctrl->annoyance_level[reason]) )
```

Comment:
What is `(annoy_lv > cctrl->annoyance_level[reason])` for here?

Reply:
So anger can still increase.

Reply:
ok